### PR TITLE
[1.x] Add profile information update custom response

### PR DESCRIPTION
**Pull request by NargesHajiani**
This PR aims to add custom response binding to profile information update feature.

### CODE REVIEW

File: src/Http/Controllers/ProfileInformationController.php [hunk 1]
```diff
@@ -14,15 +14,14 @@ class ProfileInformationController extends Controller
...
-     * @return \Illuminate\Http\Response
+     * @return \Laravel\Fortify\Contracts\ProfileInformationUpdateResponse
      */
-    public function update(Request $request,
-                           UpdatesUserProfileInformation $updater)
-    {
+    public function update(
+        Request $request,
+        UpdatesUserProfileInformation $updater
+    ) {
```
Comment (driesvints) [inline src/Http/Controllers/ProfileInformationController.php:22]: Please undo the styling changes here.

Comment (NargesHajiani) [reply src/Http/Controllers/ProfileInformationController.php]: fixed.

Comment (driesvints) [reply src/Http/Controllers/ProfileInformationController.php]: It's still incorrect. Please revert it to the original code.

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Comment by NargesHajiani** [comment]
@taylorotwell 
In update profile information feature in Fortify, unlike registration, login and update password, we can not define a custom response class, so like my previous merged PR (#290) I opened this PR to add this ability to the package.

**Pull request closed without merging**